Floofbot/Services/NicknameAlert.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Discord;
+using Floofbot.Services.Repository;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Floofbot.Services
+{
+    public class NicknameAlert
```

Comment:
I'd really prefer if this were renamed to NicknameAlertService.
I'm also not sure what this file is for.

Reply:
Can rename this in a future PR, this is just a placeholder for the service right now